### PR TITLE
S02-102-GROUP03

### PR DESCRIPTION
**Pull request by duclair3006**
# WELCOME TO DEVOPS-EASY-LEARNING PULL REQUEST SYSTEM 
# Description of the change

> Detailed description here

## Checks and balances

- [ ] Tests added and run
- [ X] (External) Interfaces documented
- [ ] Has security implications (describe below)

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing
functionality to not work as expected)

## Related stories, issues and pulls

> Links to Pivotal Tracker stories, JIRA tickets or other pull requests here

## Security considerations

> Describe any security impact this change may have

### CODE REVIEW

File: group03.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+duclair
+idrissa
+valdes
+philomene
+babara
+jonathan
+cyprien
+nigieu
+pat
```
Comment (YvanNys) [inline group03.txt:9]: why cann't put full name

**Review by YvanNys — CHANGES_REQUESTED** [review]

File: group03.txt (hunk 1 above)
Comment (idi10) [reply group03.txt:9]: last name before first name

Comment (Galiomid) [reply group03.txt:9]: It's not nigieu but Mr Nigieu

File: group03.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+duclair
+idrissa
+valdes
```
Comment (willyaldes) [inline group03.txt:3]: Why didn't you start the names with Capital letter?

**Review by willyaldes — CHANGES_REQUESTED** [review]

File: group03.txt (hunk 1 above)
Comment (idi10) [reply group03.txt:9]: Capital letters?

File: group03.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+duclair
```
Comment (pdarso) [inline group03.txt:1]: looking good

File: group03.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+idrissa
+valdes
+philomene
+babara
+jonathan
+cyprien
```
Comment (yncyprien) [inline group03.txt:7]: I think a capital C will be better.

File: group03.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
+duclair
+idrissa
+valdes
+philomene
```
[the remark this answers is not in the document]
Comment (pdarso) [reply group03.txt:4]: Thanks Mr. I tried thus

**Review by yncyprien — CHANGES_REQUESTED** [review]

File: group03.txt (hunk 1 above)
Comment (leslymegne) [reply group03.txt:9]: my name is actually Lesly not Leslie lol

**Review by pdarso — CHANGES_REQUESTED** [review]

File: group03.txt (hunk 4 above)
Comment (yncyprien) [reply group03.txt:7]: Why dont we consider Fisrt and Last names

File: group03.txt [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+babara
+jonathan
+cyprien
+nigieu
```
[the remark this answers is not in the document]
Comment (leslymegne) [reply group03.txt:8]: good

**Pull request closed without merging**